code/datums/quirks/neutral.dm

```diff
@@ -231,6 +231,14 @@
 
 	give_item_to_holder(wayfinder, list(LOCATION_LPOCKET = ITEM_SLOT_LPOCKET, LOCATION_RPOCKET = ITEM_SLOT_RPOCKET, LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
 
+/datum/quirk/shifty_eyes
+	name = "Shifty Eyes"
+	desc = "Your eyes tend to wander all over the place, whether you mean to or not, causing people to sometimes think you're looking directly at them when you aren't."
+	icon = "far fa-eye"
```

Comment:
What does this look like? I forget what the far discriminant is.

Reply:
tbh i got no clue, i just copied what the fontawesome site told me. Should I just try `eye`?

Reply:
They'll be different images, try both and see which one you like better, then post a picture of it alongside the other quirk icons.